migration/re_encryption.py

```diff
@@ -0,0 +1,205 @@
+import urllib.request
+import argparse
+import boto3
+import base64
+import cryptography
+import json
+import os
+import pg8000
+import logging
+from botocore.exceptions import ClientError
+from pprint import pprint
+
+
+class ReEncrypter:
+    aws_account_id = ''
+    aws_iam_session = ''
+    aws_kms_client = ''
+    old_pg_client_applications = ''
+    old_pg_client_cases = ''
+    new_kms_key_id = ''
+    aws_kms_client = ''
+    unique_aws_kms_keys = []
+
+    def __init__(self):
+        cases = {}
+        env_vars = [
+            "OPG_REFUNDS_DB_CASES_FULL_PASSWORD",
+            "OPG_REFUNDS_DB_CASES_FULL_USERNAME",
+            "OPG_REFUNDS_DB_CASES_HOSTNAME",
+            "OPG_REFUNDS_DB_CASES_NAME"
+        ]
+        path = "/etc/docker-compose/caseworker-api/api.env"
+        with open(path, "r") as f:
```

Comment:
Nice Exception handling? File  might not be present or readable with running user.